travis/travis_install_nightly

```diff
@@ -1,6 +1,6 @@
 #!/bin/bash
 
-pip install -q QUnitSuite flake8 coveralls pylint Click > /dev/null 2>&1
+pip install -q QUnitSuite flake8 coveralls pylint Click docutils git+https://github.com/moylop260/pylint_oca.git > /dev/null 2>&1
```

Comment:
I'm agree with you.
The code to review the pylint_oca is too much.
To avoid a big PR again, I splitted to a temporal project.
But the focus is create: 
 - Option a) New project
 - Option b) New PR to MQT
 - Option c) New PR to MT
(ToDo: Discuss it)
I will send a email to community for know POV after merge this one.